code/game/gamemodes/changeling/powers/lingstuns.dm

```diff
@@ -0,0 +1,18 @@
+/obj/effect/proc_holder/changeling/spike_puff
+	name = "Spine Burst"
+	desc = "Our pores fire a spine containing the contents of our chemical sacks in every direction, injecting those in range."
+	helptext = "Fires a burst of spines that stun anyone in melee range."
+	chemical_cost = 40 //Tentative attempt at balance
+	dna_cost = 2
+	req_human = 1
+
+obj/effect/proc_holder/changeling/spike_puff/sting_action(var/mob/user)
+	user << "<span class='notice'>We empty our chemical sack into spines, and fire them.</span>"
+	playsound(user.loc, "sound/weapons/slash.ogg" , 50)
+	for(var/mob/living/M in range(2, user))
+		if(iscarbon(M))
+			user.visible_message("<span class='warning'>[M] has been sprayed with toxic spines!</span>")
```

Comment:
Actually using visible_message here is going to spam everyone with the message as many times as there are mobs around. If we want to keep the for(var/mob/living/M) stuff we need to use show_message(). It's not very elegant but the only other way would be to add a range argument to visible_message (and that's a lot of work).